web_autoflip_demo/video_upload/video_upload.js

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
If this is an auto-gen file from ts to js, add a line to the .gitignore to remove .js files

Reply:
Sure, will do quickly!

Reply:
Hi, Justin, I added the .gitignore and to remove the .js files, please check.

Reply:
Thanks for adding that!

Reply:
No problem!